content/practices/postmortem/index.md

```diff
@@ -0,0 +1,121 @@
+   
+   > “What caused the recent incident? And how did we fix it? Additionally we want to define action items to avoid this in the future and document it in our postmortem document.  We are not looking to find someone to blame.” 
+   
+   {{< callout >}}
+   Tip: Embrace blameless culture because you want to identify what happens and how to avoid it in the future. Make sure you provide psychological safety for each team member.
+   {{< /callout >}}
+
+1. **Present the incident timeline (5 minutes)**
+
+   Show the incident timeline and confirm it with the team.
+
+1. **Brainstorming Postmortem Topic (5 minutes)**
+
+   Ask team to write on post-it or digital workspace:
+   
+   - What went wrong - What caused the recent incident?
+   - What went well - What did we do that worked well to fix the incident?
+   - Where we got lucky - Are there any events/things that helped us out?
+
+1. **Quick Topic Clustering (5 minutes)**
+   
+   Ask the team to cluster the post-its based on similar topics.
+
+1. Discuss each cluster (20 minutes)
+   
+   Allow enough time for each cluster to be discussed. If any particular clusters need additional time, consider a follow-up session about that specific topic.
+   
+   Additionally, for clusters in the “what went wrong” column, ask “Why did this happen?” and “How can we reduce the impact if it happens again?” 
+   
+   {{< callout >}}
+   Tip: consider using [“five whys” method](https://en.wikipedia.org/wiki/Five_whys). Again, be sure to focus on root causes that are due to the process, not individual blame.  
+   {{< /callout >}}
+
+1. **Discuss the action items (15 minutes)**
+
+   Discuss with the team what actions we can take to solve the incident and avoid it in the future. Define the owner(s) of these actions.
+
+1. **Agreed on the postmortem owner(s) (4 minutes)**
+
+   Decide as a team the postmortem owner, whose responsibilities are to:
+   
+   - Create a postmortem document for future reference that contains:
+      - Overview of the incident
+      - Action items to resolve the incident
+      - [Root cause analysis](https://asq.org/quality-resources/root-cause-analysis)
+   - Learnings and next steps to avoid the incident happening in the future
+   - Communicate with stakeholders regarding the status of the postmortem
+   - Plan review meeting to make sure all action items in the postmortem meeting is done
+
+1. **Schedule the follow up (1 min)**
+
+   Agree with the team for a follow up postmortem session to review the actions. The incident is not closed until all postmortem actions are completed. 
+
+## Success/Expected Outcomes
+At the end of this exercise, the team will have a clear alignment of what the incident was and how it happened. Moreover, the team will have a set of plans to prevent or reduce the impact of the incident’s recurrence. 
+
+## Facilitator Notes & Tips
+Examples of postmortem documents: 
+
+- https://sre.google/sre-book/example-postmortem/
```

Comment:
It would be great to add a description of the links instead of having to figure out what they are by clicking ;-)

In this case something like "Google SRE Example - Shakespeare Sonnet++" would do it.

Reply:
Great suggestion. Updated.